### PR TITLE
AV-188865 Secret Handling Fix

### PR DESCRIPTION
**Pull request by pkoshtavmware**
AV-188865 Secret Handling Fix

### CODE REVIEW

File: ako-gateway-api/nodes/dequeue_ingestion.go [hunk 1]
```diff
@@ -62,6 +62,11 @@ func DequeueIngestion(key string, fullsync bool) {
 			utils.AviLog.Errorf("key: %s, msg: no model found: %s", key, modelName)
 			continue
 		}
+		if objType == utils.Secret {
+			handleGateway(parentNs, parentName, fullsync, key)
```
Comment (swathinsankaran) [inline ako-gateway-api/nodes/dequeue_ingestion.go:66]: In the next release, we should revisit this logic and call the method that actually does the secret handling.

**Review by swathinsankaran — APPROVED** [review]
LGTM

**Comment by swathinsankaran** [comment]
build ako

File: ako-gateway-api/nodes/dequeue_ingestion.go (hunk 1 above)
Comment (pkoshtavmware) [reply ako-gateway-api/nodes/dequeue_ingestion.go:66]: Sure. Noted

**Comment by pkoshtavmware** [comment]
Ut results: 
Running tool: /usr/local/go/bin/go test -timeout 30s -run ^TestSecretCreateDelete$ github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/graphlayer

ok  	github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/tests/gatewayapitests/graphlayer	10.946s

**Comment by pkoshtavmware** [comment]
build ako

File: ako-gateway-api/nodes/dequeue_ingestion.go (hunk 1 above)
Comment (akshayhavile) [reply ako-gateway-api/nodes/dequeue_ingestion.go:66]: With this logic, we will work on all secret associated with that GW. Can we optimize this?

Comment (pkoshtavmware) [reply ako-gateway-api/nodes/dequeue_ingestion.go]: Optimised it and added calling for the method that does only Secret handling

**Review by aaha97 — COMMENTED** [review]
LGTM

**Review by aaha97 — APPROVED** [review]

**Comment by aaha97** [comment]
build ako

**Comment by aaha97** [comment]
build ako

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**